landing-page/types/headnode/layouts/sidebar.html.erb

```diff
@@ -36,10 +36,12 @@
       <h2>Quick Access</h2>
       <ul class="w-100">
         <% app_items.each do |app| %>
+          <% icon = "<span class=\"fa fa-solid fa-fw fa-#{app[:fa_icon]}\"></span>" %>
+          <% title = "<span class=\"title\">#{app[:short_title] || app[:title]}></span>" %>
```

Comment:
There's an extra `>` on this line (or somewhere).

![image](https://user-images.githubusercontent.com/287050/162019142-2e21b462-a9cf-414f-9f2c-eeb91f1b5aec.png)